docs/integrations/destinations/clickhouse.md

```diff
@@ -80,6 +80,7 @@ Therefore, Airbyte ClickHouse destination will create tables and schemas using t
 
 | Version | Date       | Pull Request                                               | Subject                                                             |
 |:--------|:-----------|:-----------------------------------------------------------|:--------------------------------------------------------------------|
+| 0.2.4   | 2023-06-05 | [\#27036](https://github.com/airbytehq/airbyte/pull/27036) | Internal scaffolding update                                         |
```

Comment:
Maybe we use the same changelog message for all of these: `Internal code change for future development (install normalization packages inside connector)`